util/db/interfaces.go

```diff
@@ -31,5 +32,14 @@ import (
 type Queryable interface {
 	Prepare(query string) (*sql.Stmt, error)
 	Query(query string, args ...interface{}) (*sql.Rows, error)
+	QueryContext(ctx context.Context, query string, args ...interface{}) (*sql.Rows, error)
 	QueryRow(query string, args ...interface{}) *sql.Row
+	QueryRowContext(ctx context.Context, query string, args ...interface{}) *sql.Row
+}
+
+// Executable is similar but has write methods as well.
+type Executable interface {
```

Comment:
I think this should be `Executor` , not `Executable`. So it's more like the usual Go conventional inteface name, `Writer`. The thing that implements this interface is capable of performing Execs, it cannot, itself, be "Executed".  The above interface `Queryable` is different.  There, the thing can, itself, be queried, so it is indeed Queryable.

Reply:
> By convention, one-method interfaces are named by the method name plus an -er suffix or similar modification to construct an agent noun: Reader, Writer, Formatter, CloseNotifier

I guess the name `Executable` was given after `Queryable` above and the fact it is not a single method interface.
Maybe a better name is `QueryableExecutor` :)